src/drivers/amd/rembrandt/acp_dmic_dma.c

```diff
@@ -33,6 +33,15 @@
 #include <platform/chip_registers.h>
 #include <platform/acp_dmic_dma.h>
 
+extern char *dmic_rngbuff_addr1;
+extern int *dmic_rngbuff_iaddr;
+extern uint32_t bytes_per_sample;
+extern uint32_t num_chs;
+extern uint32_t samplerate_khz;
+extern uint32_t silence_cnt;
+extern uint32_t silence_incr;
+extern int coeff;
+extern int numFilterBuffers;
```

Comment:
ok, so not sttaic as per my previous comment. This really looks like it needs to be a structure that can be part of private data for the AMD dmics.

Reply:
created structure for data.